Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py

```diff
@@ -0,0 +1,52 @@
+import demistomock as demisto  # noqa
+from CommonServerPython import *  # noqa
+import cv2
+
+
+def read_qr_code(filename: str) -> str:
+
+    detect = cv2.QRCodeDetector()
+    img = cv2.imread(filename)
+    text, *_ = detect.detectAndDecode(img)
```

Comment:
what else does this return that we are ignoring? Lets log it

Reply:
done